cloudquery-cloud/README.md

```diff
@@ -0,0 +1,38 @@
+# CloudQuery Cloud
+
+![datadog-integration][1]
+
+## Overview
+
+[CloudQuery][2] is an open-source, high-performance data integration framework built for developers, with support for a wide range of plugins.
+
+CloudQuery extracts, transforms, and loads configuration from cloud APIs to a variety of supported destinations such as databases, data lakes, or streaming platforms for further analysis.
+
+[CloudQuery Cloud][3] is a great way to get started with CloudQuery and syncing data from source to destination without the need to deploy your own infrastructure. It is also much easier to connect to sources and destinations with the integrated OAuth authentication support. You only need to select a source and destination plugin and CloudQuery will take care of the rest.
+
+## Setup
+
+### Installation
+
+Sign up for free at [cloud.cloudquery.io][3]. Once logged in, navigate to the **Sources** page and add a Datadog source. Under the **Authentication** section, use the **Authenticate** button to grant access to your Datadog account using OAuth2 flow.
```

Comment:
Could you reformat this to describe the instructions a user would take when starting the OAuth flow from Datadog? See an example [here](https://app.datadoghq.com/integrations/ably) on the configure tab.